test/OpenTelemetry.Tests/Metrics/AggregatorTest.cs

```diff
@@ -104,6 +104,46 @@ public void HistogramDistributeToAllBucketsCustom()
             Assert.Equal(boundaries.Length + 1, actualCount);
         }
 
+        [Fact]
+        public void HistogramBinaryBucketTest()
```

Comment:
I'd suggest to add e2e test for Histogram targeting the edges (default bounds, bound.length >50 etc.)